docs/REST API/rest-api.md

```diff
@@ -8548,8 +8571,67 @@ Returns information about the specified software version.
 }
 ```
 
-## Vulnerabilities
+### Get available App Store apps
+
+Returns the list of App Store (VPP) apps purchased in Apple Business Manager.
+
+`GET /api/v1/fleet/software/app_store/:team_id`
+
+#### Parameters
+
+| Name    | Type | In | Description |
+| ------- | ---- | -- | ----------- |
+| team_id | integer | query | **Required**. The team ID. Lists available VPP software for specified team. |
```

Comment:
How would the team id change the response? Wouldn't this just be the full list every time?

Or are you looking for a status in the response on if this app is already assigned to this team?

Reply:
Added a suggestion below to show what I think this means (to enable the checkboxes in the UI when loading the list)

Reply:
I was thinking of filtering a response depending on if the app is already added to a team.

Reply:
I think that makes sense. Also, it's weird that we don't list apps that are in ABM in the API.

Reply:
Apps that get added should definitely create a title if their isn't a matching one already so they should be. Or I might have misunderstood

Reply:
Sorry if I confused you. Right now idea is that this endpoint only lists VPP apps from ABM that aren't already added to a team (aren't assigned to a title). If we change this and add `added` flag, I think it's better for API users because we'll list all apps that are available in ABM, not only ones that aren't assigned to a titles.